sbin/gen-test-certs

```diff
@@ -10,11 +10,13 @@ PHRASE=${2:-MySecretPassPhrase42}
 mkdir -p $ROOT/bin/tls
 cd $ROOT/bin/tls
 
-openssl genrsa -out ca.key 4096
+[[ -f .generated && $(cat .generated) == $WITH_PASSPHRASE ]] && exit 0
+
+openssl genrsa -out ca.key 2048
 openssl req \
     -x509 -new -nodes -sha256 \
     -key ca.key \
-    -days 3650 \
+    -days 365 \
```

Comment:
Whats this? it was an error?

Reply:
No. I just don't see the point of having some certifications expire after a year and others after 10. If you didn't regenerated the certifications for a year you'll have to regenerate all of them anyway